### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pierceseeker**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+	background: rgba(0, 0, 0, 0.8);
+}
+header .fixedmenu {
+	font-family: 'Trade Winds';
+	font-size: 1em;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+}
+.fixedmenu a {
```
Comment (Kupfeev) [inline css/index.css:20]: .fixedmenu is repeated, 
Try to add everything in one go.

File: img/wikipedia.png [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="wikipedia-w" class="svg-inline--fa fa-wikipedia-w fa-w-20" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512"><path fill="currentColor" d="M640 51.2l-.3 12.2c-28.1.8-45 15.8-55.8 40.3-25 57.8-103.3 240-155.3 358.6H415l-81.9-193.1c-32.5 63.6-68.3 130-99.2 193.1-.3.3-15 0-15-.3C172 352.3 122.8 243.4 75.8 133.4 64.4 106.7 26.4 63.4.2 63.7c0-3.1-.3-10-.3-14.2h161.9v13.9c-19.2 1.1-52.8 13.3-43.3 34.2 21.9 49.7 103.6 240.3 125.6 288.6 15-29.7 57.8-109.2 75.3-142.8-13.9-28.3-58.6-133.9-72.8-160-9.7-17.8-36.1-19.4-55.8-19.7V49.8l142.5.3v13.1c-19.4.6-38.1 7.8-29.4 26.1 18.9 40 30.6 68.1 48.1 104.7 5.6-10.8 34.7-69.4 48.1-100.8 8.9-20.6-3.9-28.6-38.6-29.4.3-3.6 0-10.3.3-13.6 44.4-.3 111.1-.3 123.1-.6v13.6c-22.5.8-45.8 12.8-58.1 31.7l-59.2 122.8c6.4 16.1 63.3 142.8 69.2 156.7L559.2 91.8c-8.6-23.1-36.4-28.1-47.2-28.3V49.6l127.8 1.1.2.5z"></path></svg>
```
Comment (Kupfeev) [inline img/wikipedia.png:1]: Delete archives that you don't use in your proyect.

File: img/google.png [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="google" class="svg-inline--fa fa-google fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 488 512"><path fill="currentColor" d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"></path></svg>
```
Comment (Kupfeev) [inline img/google.png:1]: Delete archives that you don't use in your proyect.

**Review by Kupfeev — CHANGES_REQUESTED** [review]
Changes and file deletions required.

File: css/index.css (hunk 1 above)
Comment (Pierceseeker) [reply css/index.css:20]: Cambios realizados.

**Review by Kupfeev — APPROVED** [review]

**Pull request merged**